x-pack/legacy/plugins/ml/public/application/contexts/kibana/use_timefilter.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect } from 'react';
+
+import { useMlKibana } from './kibana_context';
+
+interface UseTimefilterOptions {
+  timeRangeSelector?: boolean;
+  autoRefreshSelector?: boolean;
+}
+
+export const useTimefilter = ({
+  timeRangeSelector = true,
+  autoRefreshSelector = true,
```

Comment:
should these have default values?
if i just wanted to enable one, it'll automatically turn on the other.

Reply:
Good point! I changed it in f162a64 to not do anything for the options that are not set.